CMakeLists.txt

```diff
@@ -98,8 +98,14 @@ endif()
 
 find_package(Git)
 if(GIT_FOUND AND EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/.git")
+	if(MINGW)
+		set(GDXSV_PATTERN "gdxsv-\\*")
+	else()
+		set(GDXSV_PATTERN """gdxsv-*""")
+	endif()
+	message(GDXSV_PATTERN="${GDXSV_PATTERN}")
```

Comment:
No need this line any more

Reply:
Let me try, not sure if the `\*` method works on all platform

Reply:
not working for Apple and Linux unfortunately